docs/OnnxReleases.md

```diff
@@ -78,7 +78,6 @@ The ONNX project, going forward, will plan to release roughly on a four month ca
    * https://github.com/onnx/tensorflow-onnx
    * https://github.com/onnx/sklearn-onnx
    * https://github.com/onnx/onnxmltools
-   * https://github.com/onnx/keras-onnx
    * https://github.com/onnx/onnx-tensorrt
    * https://github.com/onnx/onnx-coreml
```

Comment:
nit: https://github.com/onnx/onnx-coreml seems unmaintained for a while and also https://github.com/onnx/onnx-tensorflow will be deprecated soon. Probably we can remove them as well.

Reply:
Done.